### PR TITLE
v2 podman search rootless

### PR DESCRIPTION
**Pull request by baude**
enable the search command for rootless

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

File: cmd/podman/containers/mount.go [hunk 1]
```diff
@@ -30,9 +30,9 @@ var (
 		Args: func(cmd *cobra.Command, args []string) error {
 			return parse.CheckAllLatestAndCIDFile(cmd, args, true, false)
 		},
-		Annotations: map[string]string{
-			registry.ParentNSRequired: "",
-		},
+		//Annotations: map[string]string{
```
Comment (rhatdan) [inline cmd/podman/containers/mount.go:33]: This change does not match the PR.

Comment (baude) [reply cmd/podman/containers/mount.go:33]: duh righjt

**Comment by mheon** [comment]
/lgtm
/hold

I love easy to review PRs

**Comment by jwhonce** [comment]
LGTM

**Pull request merged**